app/matplotlib.ipynb

```diff
@@ -47,20 +48,38 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
+    "plt.figure(figsize=(6, 4))\n",
+    "plt.plot(years, population)\n",
+    "\n",
+    "plt.xticks(range(min(years), max(years) + 1, 1))\n",
```

Comment:
You don't need to create a range for xticks, just put series there, like this: plt.xticks(years)